app/client/src/features/form/components/fragments/Job.js

```diff
@@ -88,7 +88,7 @@ function Job({ highlights, index, addHighlight, removeHighlight }: Props) {
             </RoundButton>
             <RoundButton
               inverted
-              disabled={i !== highlights.length - 1}
+              disabled={highlights.length === 1 }
```

Comment:
nitpick: Could you just get rid of the trailing whitespace here? `=== 1` instead of `=== 1 `.

You can just run `npm run format` to automatically format the project, so you don't have to do any style changes yourself.

Reply:
Done